### PR TITLE
Release/0.20.0

### PR DESCRIPTION
**Pull request by mrk-han**
* Deprecate swipeDown/Up
  - Create scrollDown/Up

* Reduce log level (#152)

* Update TestNG to 7.1.0

* Update Java-Client to 7.3.0

* Fix exception throwing in getHub()
  - and autoformat

* Update TestNG to 7.1.0, Java-Client to 7.3.0 (#151)

* Deprecate swipeLeft/Right, create scrollLeft/Right
  - Add javadoc comments

* Fix swipe lengths tests

* Add default error handling to certain base methods and Add tests (#144)

* add null check for quit method (#142)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -12,7 +12,7 @@ Conductor Mobile is a port of the [Conductor](https://github.com/conductor-frame
     <dependency>
         <groupId>com.willowtreeapps</groupId>
         <artifactId>conductor-mobile</artifactId>
-        <version>0.19.2</version>
+        <version>0.19.3</version>
```
Comment (aaron-goff) [inline README.md]: Should this be 0.20?

File: build.gradle [hunk 2]
```diff
@@ -6,7 +6,7 @@ plugins {
 }
 
 group = 'com.willowtreeapps'
-version = '0.19.2'
+version = '0.19.3'
```
Comment (aaron-goff) [inline build.gradle]: Same - 0.20?

File: README.md (hunk 1 above)
Comment (mrk-han) [reply README.md]: You da best, fixed in fa56888b59e7a52855ad425b8d4ae8ee1bbe1040

File: build.gradle (hunk 2 above)
Comment (mrk-han) [reply build.gradle]: fa56888b59e7a52855ad425b8d4ae8ee1bbe1040

**Pull request merged**